_posts/2023-08-01-fedramp-csp-community.md

```diff
@@ -11,4 +11,4 @@ The goal of the community is to bring together people working on FedRAMP complia
 
 The list is open to compliance staff at CSPs listed at the [FedRAMP Marketplace](https://marketplace.fedramp.gov) as authorized or in-process, or when a CSP has retained a 3PAO to pursue authorization.
 
-You can read all about this at [cloud.gov Compliance Community](({{ site.baseurl }}{% link _docs/compliance/compliance-community.md %})), and join today by sending an email to [cloud-gov-compliance@gsa.gov](mailto:cloud-gov-compliance@gsa.gov) from the domain of your FedRAMP CSP.
+You can read all about this at [cloud.gov Compliance Community](({{ site.baseurl }}{% link _docs/compliance/compliance-community.md %})), and join today by sending an email to [community@cloud.gov](mailto:community@cloud.gov) from the domain of your FedRAMP CSP.
```

Comment:
```suggestion
You can read all about this at [cloud.gov Compliance Community]({{ site.baseurl }}{% link _docs/compliance/compliance-community.md %}), and join today by sending an email to [community@cloud.gov](mailto:community@cloud.gov) from the domain of your FedRAMP CSP.
```

Reply:
This link is broken on the page as is: https://cloud.gov/2023/08/01/fedramp-csp-community/

Reply:
And broken in the preview: https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.sites.pages.cloud.gov/preview/cloud-gov/cg-site/peterb/community-2/2023/08/01/fedramp-csp-community/